scripts/email/templates/dryrun.py

```diff
@@ -0,0 +1,5 @@
+BCC = ["hyz4@cornell.edu"]
+SUBJECT = "Sign up for CoursePlan 🔫"
```

Comment:
this is too funny

Reply:
the first test passed but at the cost of my soul 😿